packages/client/src/object/convertWireToOsdkObjects.ts

```diff
@@ -48,6 +50,42 @@ function createPrototype<
   return proto;
 }
 
+// preprocess the ontology definition to more quickly apply object conversions when needed
```

Comment:
To be nitty, is it faster to create a function that must be invoked that has to iterate over an array of steps to do more function invocations? Would it just be faster to iterate over the properties?

Reply:
If you have to examine every property for every object, that's potentially quite a bit slower than preprocessing the config -> transform and then reusing the transform over and over. Here's a vaguely representative jsbench:

![image](https://github.com/ericanderson/osdk-ts/assets/24275386/55cdc388-22de-4e65-bf53-6dd1eb1016c5)